### PR TITLE
Integration with ElevenLabs text to speech

### PR DESCRIPTION
**Pull request by mateusz-wosinski-ds**
- Description: adds integration with ElevenLabs text-to-speech [component](https://github.com/elevenlabs/elevenlabs-python) in the similar way it has been already done for [azure cognitive services](https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/tools/azure_cognitive_services/text2speech.py)
  - Dependencies: elevenlabs
  - Twitter handle: @deepsense_ai, @matt_wosinski
  - Future plans: refactor both implementations in order to avoid dumping speech file, but rather to keep it in memory.

### CODE REVIEW

File: libs/langchain/langchain/tools/eleven_labs/text2speech.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+class ElevenLabsText2SpeechTool(BaseTool):
+    """Tool that queries the Eleven Labs Text2Speech API.
+
+    In order to set this up, follow instructions at:
+    https://docs.elevenlabs.io/welcome/introduction
+    """
+
+    model: Union[ElevenLabsModel, str] = ElevenLabsModel.MULTI_LINGUAL
+
+    name: str = "eleven_labs_text2speech"
```
Comment (baskaryan) [inline libs/langchain/langchain/tools/eleven_labs/text2speech.py:27]: from tools(ie LLMs) perspective does it matter that it's eleven labs? same with mentioning eleven labs in description

File: libs/langchain/pyproject.toml [hunk 2]
```diff
@@ -129,6 +129,7 @@ markdownify = {version = "^0.11.6", optional = true}
 assemblyai = {version = "^0.17.0", optional = true}
 dashvector = {version = "^1.0.1", optional = true}
 sqlite-vss = {version = "^0.1.2", optional = true}
+elevenlabs = {version = "^0.2.24", optional = true}
```
Comment (baskaryan) [inline libs/langchain/pyproject.toml:132]: can we skip adding to pyproject? we're trying to move away from having optional dependencies in here and using extra groups, they don't add much to user experience and can cause problems with dev setup

**Comment by baskaryan** [comment]
2 small comments, otherwise looks great!

File: libs/langchain/langchain/tools/eleven_labs/text2speech.py (hunk 1 above)
Comment (mateusz-wosinski-ds) [reply libs/langchain/langchain/tools/eleven_labs/text2speech.py:27]: it is analogous to: [azure text-to-speech](https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/tools/azure_cognitive_services/text2speech.py
) component. Implemented this way to keep consistency

**Comment by mateusz-wosinski-ds** [comment]
@baskaryan pls review and let me know if it is ready to merge. I would like to refactor tts modules in the next PR

File: libs/langchain/pyproject.toml (hunk 2 above)
Comment (mateusz-wosinski-ds) [reply libs/langchain/pyproject.toml]: shouldn't I add elevenlabs to pyproject to make tests pass?

**Pull request merged**